### DISCUSSION TITLE
Issue with /webmail and /cpanel when setup the domain

### DISCUSSION THREAD

**Discussion by in2uitions** [vercel/community#2949] [Help]
### Summary

I have my domain setup on Godaddy. when i linked Vercel for the frontend i cannot access /cpanel and /webmail, it gives me 404 page.

### Example

https://riaddaoud.com/webmail
https://riaddaoud.com/cpanel

### Steps to Reproduce

A | @ | 76.76.21.21

I have only pointed A record towards vercel

**Comment by smaeda-ks** [comment]
@in2uitions I'm not very familiar with Godaddy, but you should contact Godaddy support to see if you can host cpanel/webmail on a subdomain instead of your Apex domain in which you're trying to point to Vercel to build websites.
